### DISCUSSION TITLE
Possible to have duplicate INET versions in omnet?

### DISCUSSION THREAD

**Discussion by user2038493** [omnetpp/omnetpp#1049] [Q&A] [answered]
I need to have two dupplicate versions of INET framework with different functionalities (since I have to edit the internal source code it is the easiest way I think)

If I duplicate the folders, I get a lot of errors, maybe I am using the wrong approach for this

Someone please enlighten me

**Comment by rhornig** [comment, marked as answer]
you can have two different versions in the IDE workspace, but they should not be open both at the same time.
